doc/usage/tutorial.md

```diff
@@ -2,53 +2,99 @@
 
 ```bash
 # helm install $GOPATH/src/github.com/kubernetes-sigs/kube-batch/deployment/kube-batch --namespace kube-system
 ```
 
-Note: If there is an error `Error: apiVersion "scheduling.incubator.k8s.io/v1alpha1" in kube-batch/templates/default.yaml is not available`, please update your helm to latest version and try it again.
-
 Verify the release
 
 ```bash
 # helm list
 NAME        	REVISION	UPDATED                 	STATUS  	CHART                	NAMESPACE
-dozing-otter	1       	Thu Jun 14 18:52:15 2018	DEPLOYED	kube-batch-0.4.1    	kube-system
+dozing-otter	1       	Thu Mar 29 18:52:15 2019	DEPLOYED	kube-batch-0.4.2   	 kube-system
 ```
 
-NOTE: `kube-batch` need to collect cluster information(such as Pod, Node, CRD, etc) for scheduling, so the service account used by the deployment must have permission to access those cluster resources, otherwise, `kube-batch` will fail to startup. For users who are not familiar with Kubernetes RBAC, please copy the example/role.yaml into `$GOPATH/src/github.com/kubernetes-sigs/kube-batch/deployment/kube-batch/templates/` and reinstall batch.
+## Customize kube-batch
+
+Please check following configurable fields and customize kube-batch based on your needs.
+
+### kube-batch configuraions
+| configs | default value | Descriptions |
+| ----------- | ----------- | ------ | ----------- |
+| master | "" | The address of the Kubernetes API server (overrides any value in kubeconfig) |
+| kubeconfig | "" | Path to kubeconfig file with authorization and master location information. If you run kube-batch outside cluster or for debug purpose, this is very useful |
+| scheduler-name | kube-batch | kube-batch will handle pods whose `.spec.SchedulerName` is same as scheduler-name |
+| scheduler-conf | "" | The absolute path of scheduler configuration file |
+| default-queue | default | The default queue name of the job |
+| schedule-period | Seconds | The period between each scheduling cycle |
+| leader-elect | false | Start a leader election client and gain leadership before executing the main loop. Enable this when running replicated kube-batch for high availability |
+| lock-object-namespace | Define the namespace of the lock object that is used for leader election |
+| listen-address | :8080 | The address to listen on for HTTP requests. Metrics will be exposed at this port |
+
+
+### scheduler configuration
+
+By default, scheduler will read following default configuraion as `scheduler-conf`.
+
+```yaml
+actions: "allocate, backfill"
+tiers:
+- plugins:
+  - name: priority
+  - name: gang
+- plugins:
+  - name: drf
+  - name: predicates
+  - name: proportion
+  - name: nodeorder
+```
 
-### (3) Create a Job
 
+The `actions` is a list of actions that will be executed by kube-batch in order, although the "order" maybe incorrect; the kube-batch do not enforce that. In above example, `allocate, backfill` will be executed in order by kube-batch.
```

Comment:
s/is/are/

Reply:
This is a configuration term. Using `is` should be fine.